src/database/hooks/useLocalDatabaseHook.js

```diff
@@ -74,7 +85,9 @@ const useLocalDatabaseHook = (withMigration = false) => {
     channelMember,
     chat,
     user,
-    refresh
+    otherListener,
+    refresh,
+    refreshWithId
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `otherListener`, `refresh`, and `refreshWithId` to the returned object from `useLocalDatabaseHook` increases the flexibility of this hook by allowing consumers to listen to other database changes and manually trigger refreshes when necessary. Good job!


<!-- This is an auto-generated comment by OSS CodeRabbit -->